app/controllers/shared/api.php

```diff
@@ -27,7 +27,7 @@
         $parts = explode('.', $match);
 
         if (count($parts) !== 2) {
-            throw new Exception('Too less or too many parts', 400, Exception::GENERAL_ARGUMENT_INVALID);
+            throw new Exception(Exception::GENERAL_SERVER_ERROR, "The server encountered an error while parsing the label: $label");
```

Comment:
I know it's not always possible, but try to include the following information in error messages:

[What went wrong] [Why we think it happened] [What we suggest as a next step/ a suggested fix]

Reply:
I feel like this error is missing some context. If you're a developer seeing this issue, what should you do as a next step?

Reply:
```suggestion
            throw new Exception(Exception::GENERAL_SERVER_ERROR, "The server encountered an error while parsing the label: $label. Please create an issue on GitHub to allow us to investigate further https://github.com/appwrite/appwrite/issues/new/choose");
```